pom.xml

```diff
@@ -258,6 +258,11 @@
         <artifactId>jnr-fuse</artifactId>
         <version>${jnr-fuse.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.github.stateless4j</groupId>
+        <artifactId>stateless4j</artifactId>
+        <version>2.6.0</version>
```

Comment:
this adds a new dependency. The [stateless4j](https://github.com/stateless4j/stateless4j) project seems not very actively maintained, so we might consider to vendor this dependency into our own codebase. It's not very complicated.

Reply:
Is this `com.github.stateless4j` a MUST to have in our code?

I don't recommend to add such external dependencies at all, especially those without good maintenance and large user base.